test/parallel/test-assert-deep.js

```diff
@@ -1220,3 +1220,60 @@ assert.throws(
 
   assertNotDeepOrStrict(a, b);
 }
+
+// eslint-disable-next-line node-core/crypto-check
```

Comment:
Why do we need this comment? Doesn't that show a bug in the ESLint rule?

Reply:
I don't know why, but I only added because the linter complained.